magento/pwa-studio
# Swatch Treatments

## Description

Update Swatch Treatments.

## Related Issue





Closes #1390 .

## Verification Steps

1. Go to the configurable PDP.
2. Verify the tool-tips not shown when selecting the options and new treatment comes.
3. Make sure the same working in cart.

## Screenshots / Screen Captures (if appropriate)
![image](https://user-images.githubusercontent.com/13298685/62407732-179ad880-b5db-11e9-8c89-7b9525e9efbb.png)

## Proposed Labels for Change Type/Package




## Checklist:
